pkgs/by-name/re/reprepro/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  autoreconfHook,
+  bzip2,
+  db,
+  gpgme,
+  libarchive,
+  xz,
+  zlib,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "reprepro";
+  version = "5.4.7";
+
+  src = fetchFromGitLab {
+    domain = "salsa.debian.org";
+    owner = "debian";
+    repo = "reprepro";
+    tag = "reprepro-${version}";
+    hash = "sha256-/qiN79bq5uSCyaRnsBsrFDoTRIac3Pr5JOawkkl3cSk=";
+  };
+
+  buildInputs = [
+    bzip2
+    db
+    gpgme
+    libarchive
+    xz
+    zlib
+  ];
+
+  nativeBuildInputs = [
+    autoreconfHook
+  ];
+
```

```suggestion

  doInstallCheck = true;
  nativeInstallCheckInputs = [ versionCheckHook ];

  passthru.updateScript = nix-update-script { };

```

`nix-update-script` and ensure correct version of the build